packages/contracts-periphery/contracts/foundry-tests/citizen-house/Badge.t.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.15;
+
+import { Badge } from "../../universal/citizen-house/Badge.sol";
+import { Test } from "forge-std/Test.sol";
+
+contract TBadge is Badge {
+    constructor() Badge("example", "ex", "example.com") {}
+}
+
+contract BadgeTest is Test {
+    TBadge internal badge;
+
+    address testAdr1;
+
+    function setUp() public {
+        badge = new TBadge();
+        testAdr1 = 0x0000008735754EDa8dB6B50aEb93463045fc5c55;
```

Comment:
Any reason to assign this in `setUp` rather than assign it above where `testAdr1` is defined?

Reply:
Perhaps using a `foundry/forge-std` helper would be useful for creating test addresses since it creates the addr as well as labels it
https://github.com/foundry-rs/forge-std/blob/2c7cbfc6fbede6d7c9e6b17afe997e3fdfe22fef/src/Test.sol#L97